docs/worker-node/install-singularity.md

```diff
@@ -137,7 +137,10 @@ After singularity is installed, as an ordinary user run the following
 command to verify it:
 
 ```console
-user@host $ singularity exec -c --ipc --pid -H $HOME:/srv /cvmfs/singularity.opensciencegrid.org/opensciencegrid/osgvo:el6 \
+user@host $ singularity exec -c --ipc --pid \
+                --home $PWD:/srv \
+                --pwd /srv \
```

Comment:
I don't want to recommend --pwd even though the VOs are using it, because it is redundant; --home already sets the pwd there.